source/default/_patterns/00-protons/legacy/css/components/navigation/_breadcrumb.css

```diff
@@ -30,8 +30,12 @@
 	display: none;
 }
 .block--system-breadcrumb-block {
-    margin-top: -2rem;
+    margin-top: .5rem;
+	margin-bottom: 1.5rem;
 }
+/*this is a reverse of the themag_layout custom module css class in section.css*/
+.themag-layout--my-default{margin-bottom:0rem!important}
```

Comment:
This is one is a global change and it might affect the page layout in other places besides the breadcrumb.